src/idl/src/parser.h

```diff
@@ -34,8 +34,8 @@
 /* Undocumented macros, especially those whose name start with YY_,
    are private implementation details.  Do not rely on them.  */
 
-#ifndef YY_IDL_YY_SRC_PARSER_H_INCLUDED
-# define YY_IDL_YY_SRC_PARSER_H_INCLUDED
+#ifndef YY_IDL_YY_HOME_TMIEDEMA_PROJECTS_WORKSPACE_CYCLONEDDS_CYCLONEDDS_SRC_IDL_SRC_PARSER_H_INCLUDED
```

Comment:
Just checking you don't mind your local paths to be included

Reply:
I've switched the bison target to a relative path which make sure the path is not visible any longer.